build.gradle.kts

```
@@ -0,0 +1,189 @@
+    commandLine(args)
+}
+
+tasks.compileJava {
+    if (DefaultNativePlatform.getCurrentOperatingSystem().isLinux) {
+        finalizedBy("compileNative")
+    }
+}
+
+license {
+    include("**/*.java")
+    exclude("**/lz4/*.java")
+    header(rootProject.file("COPYRIGHT_HEADER.txt"))
+}
+
+val repoUrl: String = project.properties["repoUrl"] as? String ?: "https://maven.pkg.github.com/odnoklassniki/one-nio"
+
+jreleaser {
+    signing {
+        active = Active.ALWAYS
+        armored = true
+        verify = true
+    }
+    release {
+        github {
+            tagName = semVer
+            releaseName = "Release $semVer"
+            draft = true
+            sign = true
+            branch = "master"
+            branchPush = "master"
+            overwrite = true
+        }
+    }
+    deploy {
+        maven {
+            nexus2 {
+                create("maven-central") {
+                    active = Active.ALWAYS
+                    url = "https://oss.sonatype.org/service/local"
+                    stagingRepository(layout.buildDirectory.dir("staging-deploy").get().toString())
+                    setAuthorization("Basic")
+                    sign = true
+                    checksums = true
+                    sourceJar = true
+                    javadocJar = true
+                    closeRepository = true
+                    releaseRepository = false
+                }    
+            }
+        }
+    }
+}
+
+publishing {
+    publications {
+        register<MavenPublication>("release") {
+            from(components["java"])
+
+            groupId = "ru.odnoklassniki"
+            artifactId = project.name
+            addPom()
+        }
+    }
+
+    repositories {
+        maven {
+            setUrl(layout.buildDirectory.dir("staging-deploy"))
+        }
+    }
+}
+
+fun MavenPublication.addPom() {
+    pom {
+        packaging = "jar"
+        name.set("ru.odnoklassniki")
+        description.set("Unconventional Java I/O library")
+        url.set("https://github.com/odnoklassniki/one-nio")
+        issueManagement {
+            url.set("https://github.com/odnoklassniki/one-nio/issues")
+        }
+        scm {
+            connection.set("scm:git:git@github.com:odnoklassniki/one-nio.git")
+            developerConnection.set("scm:git:git@github.com:odnoklassniki/one-nio.git")
+            url.set("https://github.com/odnoklassniki/one-nio")
+        }
+        licenses {
+            license {
+                name.set("The Apache License, Version 2.0")
+                url.set("https://www.apache.org/licenses/LICENSE-2.0.txt")
+            }
+        }
+        developers {
+            developer {
+                id.set("apangin")
+                name.set("Andrei Pangin")
+                email.set("noreply@pangin.pro")
+            }
+            developer {
+                id.set("incubos")
+                name.set("Vadim Tsesko")
+                email.set("incubos@yandex.com")
```

It's time for update :)
```suggestion
                email.set("mail@incubos.org")
```